components/card/src/vwc-card.ts

```diff
@@ -46,13 +46,13 @@ export class VWCCard extends LitElement {
 
 	@property({
 		reflect: true,
-		attribute: 'supporting-text',
+		attribute: 'card-text',
 		type: String
 	})
 		supportingText: string | undefined;
 
 	private headerIconSlottedItems?: Node[];
-	private shouldShowActionsSlot: boolean | undefined;
+	private shouldShowFooterSlot: boolean | undefined;
```

Comment:
```suggestion
	#shouldShowFooterSlot: boolean | undefined;
```

Reply:
I'll change it in the file cause it needs to be change all around its mentions